pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -207,14 +207,14 @@ func init() {
 		Rules: []rbac.PolicyRule{
 			rbac.NewRule("get", "create", "update").Groups(networkGroup, legacyNetworkGroup).Resources("clusternetworks").RuleOrDie(),
 			rbac.NewRule("get", "list").Groups(networkGroup, legacyNetworkGroup).Resources("egressnetworkpolicies").RuleOrDie(),
-			rbac.NewRule("get", "list", "create", "delete").Groups(networkGroup, legacyNetworkGroup).Resources("hostsubnets").RuleOrDie(),
-			rbac.NewRule("get", "list", "create", "update", "delete").Groups(networkGroup, legacyNetworkGroup).Resources("netnamespaces").RuleOrDie(),
+			rbac.NewRule("get", "list", "watch", "create", "delete").Groups(networkGroup, legacyNetworkGroup).Resources("hostsubnets").RuleOrDie(),
+			rbac.NewRule("get", "list", "watch", "create", "update", "delete").Groups(networkGroup, legacyNetworkGroup).Resources("netnamespaces").RuleOrDie(),
 			rbac.NewRule("get", "list").Groups(kapiGroup).Resources("pods").RuleOrDie(),
 			rbac.NewRule("list").Groups(kapiGroup).Resources("services").RuleOrDie(),
 			rbac.NewRule("list").Groups(kapiGroup).Resources("namespaces").RuleOrDie(),
 			rbac.NewRule("get").Groups(kapiGroup).Resources("nodes").RuleOrDie(),
```

Comment:
That's no good. The master needs "list" and "watch" on nodes so that it can create/delete hostsubnets as nodes are created/deleted. (Um... and I have no explanation for how networking-minimal could pass with this being wrong... Is the SDN possibly picking up permissions from somewhere else? Oh, we use a SharedInformer here now, is that it?)